jstz_cli/src/accounts/account.rs

```diff
@@ -0,0 +1,64 @@
+use anyhow::Result;
+use jstz_crypto::{hash::Blake2b, public_key::PublicKey, public_key_hash::PublicKeyHash};
+use serde::{Deserialize, Serialize};
+use tezos_crypto_rs::bls::keypair_from_ikm;
+
+fn create_address(pk: PublicKey) -> String {
+    PublicKeyHash::try_from(&pk).unwrap().to_base58()
+}
+
+// Represents an individual account
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Account {
+    alias: String,
+    address: String,
+    secret_key: String,
+    public_key: String,
```

Comment:
These could match their Rust type counterparts for both improved serialization + less conversions between strings and `PublicKey`/`SecretKey`